src/docs/teams/debugger/debugging-a-transaction.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Teams | Debugging a Transaction
+layout: docs.hbs
+---
+# Debugging a Transaction
+
+<p class="alert alert-info">
+<strong>Note</strong>: Truffle Teams currently only allows you to debug transactions on <a href="/docs/teams/sandboxes/sandboxes-overview">Ganache sandboxes</a>. Support for public networks (such as the <code>Görli testnet</code> or <code>mainnet</code>) will be available soon.
```

Comment:
```suggestion
<strong>Note</strong>: Truffle Teams currently only allows you to debug transactions on <a href="/docs/teams/sandboxes/sandboxes-overview">Ganache sandboxes</a>. Support for public networks (such as the <code>Görli</code> testnet or <code>mainnet</code>) will be available soon.
```

Reply:
i'll go ahead and commit this (it was actually initially how I'd highlighted it), although it kinds looks slightly off given we're highlighting `mainnet`...minor thing though 🤷‍♂️

Reply:
It would be acceptable to not highlight either in this case